docker/src/test/java/org/jclouds/docker/features/ImageApiLiveTest.java

```diff
@@ -38,18 +49,40 @@ public void testCreateImage() {
 
    @Test(dependsOnMethods = "testCreateImage")
    public void testInspectImage() {
-      assertNotNull(api.getImageApi().inspectImage(String.format("%s:%s", DEFAULT_IMAGE, DEFAULT_TAG)));
+      image = api.getImageApi().inspectImage(String.format("%s:%s", DEFAULT_IMAGE, DEFAULT_TAG));
+      assertNotNull(image);
    }
 
    @Test(dependsOnMethods = "testInspectImage")
+   public void testTagImage() {
+      boolean tagCreated = api.getImageApi().tagImage(image.id(), "jclouds", "testTag", true);
+      assertTrue(tagCreated, "Creating new tag on Docker image should succeed.");
+      Image taggedImage = api.getImageApi().inspectImage("jclouds:testTag");
+      assertEquals(taggedImage.id(), image.id(), "Newly added image tag should point to the same image ID.");
+   }
+
+   @Test(dependsOnMethods = "testTagImage")
    public void testListImages() {
-      assertNotNull(api().listImages());
+      List<ImageSummary> listImages = api().listImages();
+      assertNotNull(listImages);
+
+      ImageSummary taggedImage = Iterables.find(listImages, new Predicate<ImageSummary>() {
+         @Override
+         public boolean apply(ImageSummary input) {
+            return input.repoTags().contains("jclouds:testTag");
+         }
+      });
+      assertNotNull(taggedImage);
    }
 
    @Test(dependsOnMethods = "testListImages")
```

Comment:
Use `alwaysRun = true` to remove the images even if the "listImages" test fails?